redhat-appstudio/service-provider-integration-operator
# TokenStorage acceptance test

### What does this PR do?
 - minor cleanup of code around tokenstorage
 - tokenstorage acceptance test run with vault and inmemory storages, see `integration_tests/tokenstorage_acceptance_test.go`
 - removes secret token storage


### Screenshot/screencast of this PR

related:
 - https://issues.redhat.com/browse/SVPI-350
 - https://issues.redhat.com/browse/SVPI-342

### What issues does this PR fix or reference?



### How to test this PR?


```
quay.io/redhat-appstudio/pull-request-builds:spi-operator-#
quay.io/redhat-appstudio/pull-request-builds:spi-oauth-#
```

There are no functional changes in production code. Passing tests should be enough.
